src/components/HeaderNavMenu.jsx

```diff
@@ -30,8 +52,27 @@ function HeaderNavMenu() {
           </MuiLink>
         </Grid>
       ))}
-      <Grid item>
-        <LoginButton className={classes.menuItem} />
+      <Grid item className={classes.avatarMenu}>
+        {(loading && <null />) ||
+          (isSignedIn ? (
+            <ProfileMenu
+              signOut={signUserOut}
+              name={user.displayName}
+              email={user.email}
+              photo={user.photoURL}
```

Comment:
Now that I think about it, I wonder why `ProfileMenu` takes in user date it can read itself from `firebase.auth()` 🤔 . Wanna try doing that?

Reply:
Yeah sure ill do that

Reply:
ok done